### PR TITLE
[ADD] AGM precursor intensity information (fragment annotation)

### PR DESCRIPTION
**Pull request by oliveralka**
The sorting function relies on precursor intensity information. 

Adding monoisotopic precursor intensity information to CompoundInfo solves the issue for fragment annotated spectra. 

Now assay library generation picks consensus spectrum from the highest intensity precursor (as expected).

### CODE REVIEW

File: src/openms/include/OpenMS/ANALYSIS/ID/SiriusMSConverter.h [hunk 1]
```diff
@@ -64,6 +64,7 @@ namespace OpenMS
     double pmass;
```
Comment (timosachsenberg) [inline src/openms/include/OpenMS/ANALYSIS/ID/SiriusMSConverter.h]: I think these structs should at least have some //< comments for the individual fields

**Comment by timosachsenberg** [comment]
can you add a test for that?

**Comment by oliveralka** [comment]
@timosachsenberg 
I will add the comments and try to find a small test case.

File: src/openms/include/OpenMS/ANALYSIS/ID/SiriusMSConverter.h [hunk 2]
```diff
@@ -50,32 +50,34 @@ namespace OpenMS
...
-    String sf_path;
-    String sf_type;
-    String sf_accession;
-    String native_id_accession;
-    String native_id_type;
+    String sf_path; //< sourcefile path for mztab-m
+    String sf_type; //< sourcefile type for mztab-m
+    String sf_accession; //< sourcefile accessions for mztab-m
+    String native_id_accession; //< nativeID accession for mztab-m
+    String native_id_type; //< nativeID type for mztab-m
```
Comment (hroest) [inline src/openms/include/OpenMS/ANALYSIS/ID/SiriusMSConverter.h]: that should be '///<`, see http://www.doxygen.nl/manual/docblocks.html#memberdoc

**Comment by oliveralka** [comment]
pyopenms_test_memoryleaktests seems unrelated

**Pull request merged**